data/initrd/scripts/remote_log_setup

```diff
@@ -0,0 +1,44 @@
+#! /bin/bash
+#Requires the LOGHOST variable to be set
```

Comment:
Given that the only "dynamic" piece of information is the loghost IP address, this will be stored in an additional file. Also we will ship a suse specific rsyslog.conf.